src/main/java/thePackmaster/vfx/calamitypack/EssenceGraspEffect.java

```diff
@@ -0,0 +1,44 @@
+package thePackmaster.vfx.calamitypack;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.vfx.AbstractGameEffect;
+
+public class EssenceGraspEffect extends AbstractGameEffect {
```

Comment:
These effects are edited versions of what Hemokinesis does -- black tendrils that reach out to the enemy.